test/fixtures/simple_example/variables.tf

```diff
@@ -36,12 +36,30 @@ variable "names" {
 }
 
 variable "per_folder_admins" {
-  type        = map(string)
-  description = "List of IAM-style members per folder who will get extended permissions."
+  type = map(object({
+    member = string
+    roles  = optional(list(string))
+  }))
+  description = "List of IAM-style roles per members per folder who will get extended permissions."
   default = {
```

Comment:
nit: can we directly just set this in the example so its easier to understand for users

Reply:
sure

Reply:
sure